cmd/buildctl/build.go

```diff
@@ -277,13 +277,26 @@ func buildAction(clicontext *cli.Context) error {
 				close(w.Status())
 			}
 		}()
+
+		solveAttr := map[string]string{}
+		frontendAttr := map[string]string{}
+		for k, v := range solveOpt.FrontendAttrs {
+			if strings.HasPrefix(k, "attest:") || strings.HasPrefix(k, "build-arg:BUILDKIT_ATTEST_") {
+				frontendAttr[k] = v
+			} else {
+				solveAttr[k] = v
+			}
+		}
```

Comment:
I think the logic for this needs to be updated in [client/build.go](https://github.com/moby/buildkit/blob/master/client/build.go#L24-L27) as well? We could modify [`attestations.Filter`](https://github.com/moby/buildkit/blob/master/frontend/attestations/parse.go#L20-L33) to return two maps, one with the attestations, and other without.